test/decompile/basic.txt

```diff
@@ -91,7 +133,31 @@ export function f(a:int, b:int):int {
     }
     if (1) continue L_b;
   }
-  return 1;
+  select_if(1, 2, 1 == 1);
+  block B_e {
+    block B_f {
+      block B_g {
+        block B_h {
+          0;
+          block B_i {
+            0;
+            br_table[B_f, B_g, B_h, B_i, ..B_e](a);
```

Comment:
Surprisingly readable already. An improvement over wat at least.
Dealing with 1000-wide br_tables will need something more custom, if that's possible to make nice at all. I'm thinking we can only nice-ify big br_tables if they have lots of repetition, something like:

```
switch N:
  cases 0-45:
    foo()
  case 46:
    bar()
  cases 47-99:
    foo() again
```

which I think is how I usually see 1K br_tables in the wild.

Reply:
One of the next PRs will change this syntax :)